time-series/time-series-api/src/main/java/com/powsybl/timeseries/FromStoreTimeSeriesNameResolver.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.timeseries;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ *
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class FromStoreTimeSeriesNameResolver implements TimeSeriesNameResolver {
+
+    private final ReadOnlyTimeSeriesStore store;
+
+    private final int version;
+
+    public FromStoreTimeSeriesNameResolver(ReadOnlyTimeSeriesStore store, int version) {
+        this.store = Objects.requireNonNull(store);
+        this.version = version;
```

Comment:
Check that version is a positive integer

Reply:
Done